src/Makefile

```diff
@@ -11,6 +11,21 @@ ARCH := $(shell uname -m | sed 's/x86_64/x86/')
 
 APPS = minimal bootstrap
 
+# Get Clang's default includes on this system. We'll explicitly add these dirs
+# to the includes list when compiling with `-target bpf` because otherwise some
+# architecture-specific dirs will be "missing" on some architectures/distros -
+# headers such as asm/types.h, asm/byteorder.h, asm/socket.h, asm/sockios.h,
+# sys/cdefs.h etc. might be missing.
+#
+# Use '-idirafter': Don't interfere with include mechanics except where the
+# build would have failed anyways.
+define get_clang_sys_includes
```

Comment:
there is no need to define the function, you can just assign this to CLANG_BPF_SYS_INCLUDES

Reply:
done, thanks